### PR TITLE
Deep learning terms

### PR DESCRIPTION
**Pull request by dsmits**
Added terms related to the deep learning workshop

### CODE REVIEW

File: glossary.yml
Comment (svenvanderburg) [inline glossary.yml]:
```suggestion
    - ml
```
Did you mean machine learning?

File: glossary.yml
Comment (svenvanderburg) [inline glossary.yml:10182]:
```suggestion
    Class imbalance refers to the problem in [machine learning](machine_learning) where there is an 
```

**Review by svenvanderburg — APPROVED** [review]
Looks good @dsmits ! I added 2 small comments.

**Review by zkamvar — APPROVED** [review]
LGTM, The only thing left is https://github.com/carpentries/glosario/pull/584#discussion_r1196387875, which needs to be addressed by @dsmits because the PR does not enable edits from maintainers.

Once that's resolved we'll merge

**Comment by elletjies** [comment]
@all-contributors please add @dsmits for translation

**Pull request merged**
